docs/source/howto/data.rst

```diff
@@ -260,6 +260,359 @@ Finding data
 Organizing data
 ===============
 
+Grouping sets of nodes together
+-------------------------------
+
+AiiDA's database is great for automatically storing all your data, but sometimes it can be tricky to navigate this flat data store.
+To create some order in this mass of data, you can *group* sets of nodes together, just as you would with files in folders on your filesystem.
+A folder, in this analogy, is represented by the :py:class:`aiida.orm.groups.group.Group` class.
+Each Group instance can hold any amount of nodes and any node can be contained in any number of nodes.
+A typical use case is to store all nodes that share a common property in a single group.
+
+Below we profide a list of typical operations that may be performed with groups.
+
+Create a new Group
+^^^^^^^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+    $ verdi group create test_group
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = Group(label='test_group')
+
+    In [2]: group.store()
+    Out[2]: <Group: "test_group" [type core], of user xxx@xx.com>
+
+
+List available Groups
+^^^^^^^^^^^^^^^^^^^^^
+Example:
+
+.. code-block:: console
+
+    $ verdi group list
+
+
+
+Groups come in different types, indicated by their type string.
+
+
+By default ``verdi group list`` only shows groups of the type *core*.
+In case you want to show groups of another type use ``-T/--type-string`` option.
+If you want to show groups of all types, use the ``-a/--all-types`` option.
+
+To filter for a specific group type, use the ``-T/--type-string`` flag:
+
+.. code-block:: console
+
+    $ verdi group list -T core.auto
+
+When using the ``QueryBuilder``, use the ``filters`` keyword to filter for a particular ``type_string``:
+
+.. code-block:: ipython
+
+    In [1]: qb = QueryBuilder().append(Group, filters={'type_string': 'core'})
+    Out[1]: <aiida.orm.querybuilder.QueryBuilder at 0x7f20db413ef0>
+
+    In [2]: qb.all(flat=True)
+    Out[2]:
+    [<Group: "another_group" [type core], of user xxx@xx.com>,
+    <Group: "old_group" [type core], of user xxx@xx.com>,
+    <Group: "new_group" [type core], of user xxx@xx.com>]
+
+Add nodes to a Group
+^^^^^^^^^^^^^^^^^^^^
+Once the ``test_group`` has been created, we can add nodes to it.
+To add the node with ``pk=1`` to the group we need to do the following.
+
+From the command line interface:
+
+.. code-block:: console
+
+    $ verdi group add-nodes -G test_group 1
+    Do you really want to add 1 nodes to Group<test_group>? [y/N]: y
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='test_group')
+
+    In [2]: from aiida.orm import Dict
+
+    In [3]: p = Dict().store()
+
+    In [4]: p
+    Out[4]: <Dict: uuid: 09b3d52a-d0c4-4e3c-823c-6157f84af920 (pk: 2)>
+
+    In [5]: group.add_nodes(p)
```

Comment:
```suggestion
    In [1]: group.add_nodes(load_node(pk=1))
```
This is not a tutorial. We only have to show the absolute minimum needed to convey how this works.